include/CppUTest/JUnitTestOutput.h

```diff
@@ -74,6 +74,9 @@ class JUnitTestOutput: public TestOutput
     virtual void writeFailure(JUnitTestCaseResultNode* node);
     virtual void writeFileEnding();
 
+private:
+    ConsoleTestOutput* currentConsoleOutput_;
+    ConsoleTestOutput defaultConsoleOutput_;
```

Comment:
I see. Here, the use case is that I change JUnitTestOutput, so it provides a way for the existing constructor to work, as well as the one that passes a console output.

I guess I'm not seeing the whole picture because it hasn't been integrated yet with the CLR.